### DISCUSSION TITLE
How can I configure mediumZoom?

### DISCUSSION THREAD

**Discussion by MonoLogueChi** [vuepress/core#1234] [Q&A]
I have a question, the plugin @vuepress/plugin-medium-zoom built in vuepress2，when I want to configure it use

[example](https://github.com/u2sb/vuepress-plugin-smplayer/blob/94200105b81c17ef6bbd2a98cf919ae8b9a06f92/docs/.vuepress/config.ts)

```ts
import { defineUserConfig } from "vuepress";
import { mediumZoomPlugin } from "@vuepress/plugin-medium-zoom";

export default defineUserConfig({
  plugins: [
    mediumZoomPlugin({
      selector: ":not(a) > img :not(.audio__player)",
    }),
  ],
});
```

warning

```
warning plugin @vuepress/plugin-medium-zoom has been used multiple times, only the last one will take effect
```

I found typeof `themePlugins.mediumZoom` is boolean，Is it supposed to be like this?

```ts
import { defineUserConfig, defaultTheme } from "vuepress";
import { mediumZoomPlugin } from "@vuepress/plugin-medium-zoom";

export default defineUserConfig({
  theme: defaultTheme({
    themePlugins: {
      mediumZoom: false,
    },
  }),
  plugins: [
    mediumZoomPlugin({
      selector: ":not(a) > img :not(.audio__player)",
    }),
  ],
});
```

This is too troublesome

**Comment by meteorlxy** [comment]
In fact, you can safely ignore that warning. It's only to make you be aware of that. Maybe we can consider downgrade it to `info` level.

**Reply by Mister-Hope** [reply]
I think warning level is ok, as we are actually resulting one plugin usage being dropped. This should be something that users should take extra attention.

**Reply by Mister-Hope** [reply]
In fact, I even add an extra check at my theme to prevent noob users calling theme built-in plugins with options which disabling features that theme requires:

https://github.com/vuepress-theme-hope/vuepress-theme-hope/blob/363adb67b00f92e06eef38e5f57b6017f079e74d/packages/theme/src/node/plugins/checkPlugin.ts#L43-L78
